### PR TITLE
Feature capstone project

### PR DESCRIPTION
**Pull request by carreraprogrammer**
For this project, I created a web page for Camp music:

- I used bootstrap and HTML for the basic structure
- I used Sass to display the mobile first and then, I used media queries for the desktop version
- I create events with JavaScript for the teachers grid and the burger menu

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @carreraprogrammer, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a linter error. please fix it and submit for the project again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by carreraprogrammer** [comment]
Hi @roshan-bajgain,

Like I said in my message when I sent the pull request, the linters that Microverse is giving me are not actualized. They are giving me linter errors with the @mixins and the @include properties of sass, do you have a solution for this?

I was reading this forum: https://questions.microverse.org/t/how-to-fix-mixin-error/3457

File: README.md [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+- Major languages: Html, scss, javascript
+- Libraries: bootstrap
 - Frameworks: Visual studio
 - Technologies used
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://carreraprogrammer.github.io/carrera_portfolio/)
-​
+- [Live Demo Link](https://carreraprogrammer.github.io/carrera_portfolio/)
```
Comment (danielufeli) [inline README.md]: - [ ] Nice work done on your project, kindly fix the link to the Live version of your project is going to your portfolio page instead.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @carreraprogrammer,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @carreraprogrammer,
### I'm Levy, your PR reviewer

Good job so far! :+1: 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Status: REQUESTED CHANGES :recycle: 
_Check the comments under the review._

 - [ ] As requested by the previous reviewer, kindly make sure that your Live Demo link is not broken. You can take a step back and follow this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/deploy/lesson_deploy_github_pages.md) to deploy your app.  :wink: 
 
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [OPTIONAL] Kindly make sure It's possible to navigate back to home, there's a link from `About` page to `Home` page :wink: 

- [OPTIONAL] Kindly make sure your first speaker photo path is accessed correctly in order to be displayed on the page.

    -  ### Your design
   image

- [OPTIONAL] Kindly make sure to stick to provided design on the mobile version.

   - ### Your design
 
   image

  - ### Provided design
   image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @carreraprogrammer         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**